src/components/SignOut/Signout.tsx

```diff
@@ -1,15 +1,13 @@
 
 import React from 'react';
-import ApiClient from '../Auth/apiClient'; 
+import ApiClient from '../Auth/apiClient';
+import { Heading } from '@chakra-ui/react';
 
 export default function HomePage() {
-    ApiClient.signout() 
+  ApiClient.signout()
   return (
-    <div className="App"> 
-      <header className="App-header">
-        <h3>Logged out</h3>
-        
-      </header>
-    </div>
-  ); 
+    <Heading>
+      Logged out
```

Comment:
To match Sign Out maybe?
```suggestion
      Logged Out
```